core/modules/user/user.tokens.inc

```diff
@@ -40,6 +40,10 @@ function user_token_info() {
     'description' => t('The user roles associated with the user account.'),
     'type' => 'array',
   );
+  $user['link'] = array(
+    'name' => t("Profile link"),
+    'description' => t("Link to account profile page based on the <em>View user profiles</em> permission."),
```

Comment:
After reading the names and descriptions of the other tokens in this function, I think the following wording would be better to use:

```suggestion
    'name' => t('Name (linked)'),
    'description' => t("The login name of the user account, displayed as a link to the user's profile page."),
```

Reply:
I've updated these according to additional feedback in the issue.